scripts/run_Cu78.jl

```diff
@@ -1,4 +1,6 @@
+
```

Comment:
It would be good if name of script makes it clear that it is for EAM?

Reply:
This works for RuNNer or the EAM as both potentials are defined.